portal-web/test/functional/com/liferay/portalweb/macros/MessageboardsPortlet.macro

```diff
@@ -308,7 +308,7 @@
 
 		<var name="key_siteName" value="${userScreenName}" />
 
-		<execute function="AssertTextEquals" locator1="SitePages#SITE_NAME" value1="${userScreenName}" />
+		<execute function="AssertTextEquals#assertPartialText" locator1="SitePages#SITE_NAME" value1="${userScreenName}" />
```

Comment:
@acbbennett Can you change the title of this commit? You're not changing a locator here, but a function.